parm/config/gfs/config.resources

```diff
@@ -454,9 +461,8 @@ case ${step} in
     export layout_x
     export layout_y
 
-    walltime="00:30:00"
     ntasks=$(( layout_x * layout_y * 6 ))
-    threads_per_task=1
+    threads_per_task=5
```

Comment:
Can we now run JEDI applications with multiple threads?

Reply:
No, this is my 'lazy' way of reducing PEs per node

Reply:
Could this be set in the `"C1152" | "C768")` case above, and 1 for all other cases?  This should be revisited for each machine at a later time using the `tasks_per_node` parameter.  I'll make a suggestion to that effect.

Reply:
@DavidHuber-NOAA does this look better?